ydb/core/kqp/ut/scheme/kqp_scheme_ut.cpp

```diff
@@ -3366,6 +3366,132 @@ Y_UNIT_TEST_SUITE(KqpScheme) {
         }
     }
 
+    Y_UNIT_TEST(CreateUserWithHash) {
```

Comment:
Can you write test where create user via SQL request and then make login operation?